articles/quickstart/backend/aspnet-core-webapi/files/configure-middleware.md

```diff
@@ -1,25 +1,21 @@
 ---
-name: Startup.cs
+name: Program.cs
 language: csharp
 ---
 
 ```csharp
-public void ConfigureServices(IServiceCollection services)
+builder.Services.AddAuthentication(JwtBearerDefaults.AuthenticationScheme)
```

Comment:
Can we add what builder is here ? So let's add `var builder = WebApplication.CreateBuilder(args);`

Reply:
Done as per [45e34bb](https://github.com/auth0/docs/pull/10231/commits/45e34bbc83210bdde43c5cbaf4c91899710e3a42).